performance-tests/bench_2/builder/Participant.cpp

```diff
@@ -110,4 +111,20 @@ bool Participant::enable(bool throw_on_error) {
     publishers_->enable(throw_on_error);
 }
 
+void Participant::detach_listeners() {
```

Comment:
```suggestion
void Participant::detach_listeners()
{
```
Not high priority, but it looks like some effort when in to style consistency elsewhere in this PR, so might as well apply here too.